EIPS/eip-1559.md

```diff
@@ -38,7 +38,7 @@ An important aspect of this fee system is that miners only get to keep the gas p
 The transition to this gas price system will occur in two phases, in the first phase both legacy and EIP1559 transactions will be accepted by the protocol. Over the course of this first phase the amount of gas available for processing legacy transactions will decrease while the amount of gas available for processing EIP1559 transactions will increase, moving gas from the legacy pool into the EIP1559 pool until the legacy pool is depleted and the EIP1559 pool contains the entire gas maximum. After all of the gas has transitioned to the EIP1559 pool, the second, finalized, phase is entered and legacy transactions will no longer be accepted on the network.
 
 ## Specification
-*Note: `//` in Python is integer division with rounding toward infinity (not truncation).*
+*Note: `//` in Python is integer division with truncation (round toward zero).*
```

Comment:
This still isn't quite correct. It rounds to negative infinity when the result is negative, it rounds to 0 when the result is positive. This has been confirmed in a Python interpreter.

Reply:
Bleh, I must he been drunk when I edited this last.  I know what the right comment is but for some reason when I type it my fingers refuse (I have failed on the comment 3 times now).  I'm AFK for the night but will fix this in the morning (hopefully 4th time is a charm).

Reply:
Fixed it to read that Python rounds toward negative infinity.  I'm typing out this description to help ensure that my fingers didn't deceive me again:
```
>>> 9/4
2.25
>>> 9//4
2
>>> # 2 is closer to negative infinity than 3
>>> (-9)/4
-2.25
>>> (-9)//4
-3
>>> # -3 is closer to negative infinity than -2
```

I am in favor of ETH 2.x's strategy of "never use negative integers anywhere in the code", but I'm not sure how to write it well for round toward infinity (it is a bit easier for round toward zero behavior, which would be consensus incompatible).  Once we are ready for a consensus change, I would like to change the comment to the following because I think it is simpler to understand if you don't have to worry about negative numbers:
> `//` is the integer division operator, which truncates

And then change the code to:
```
# check if the base fee is correct
if parent_gas_used >= parent_gas_target:
	excess_gas_used = parent_gas_used - parent_gas_target
	expected_base_fee = parent_base_fee + parent_base_fee * excess_gas_used // parent_gas_target // BASE_FEE_MAX_CHANGE_DENOMINATOR
else:
	extra_gas_available = parent_gas_target - parent_gas_used
	expected_base_fee = parent_base_fee - parent_base_fee * extra_gas_available // parent_gas_target // BASE_FEE_MAX_CHANGE_DENOMINATOR
assert expected_base_fee == block.base_fee, 'invalid block: base fee not correct'
```